add TDengine Cloud PrivateLink Help (#21493)
taosdata/TDengine: docs/en/14-network/private-link/index.md

1. Sign in to the [AWS Management Console](https://aws.amazon.com/console/) and open the Amazon VPC console at [AWS VPC](https://console.aws.amazon.com/vpc/).
2. Select the region where your VPC is located from the drop-down list in the upper-right corner. Find **Virtual private cloud** in the left navigation pane and Click **Endpoints**, and then click **Create Endpoint** in the upper-right corner. The **Create endpoint** page is displayed.

   ![TDengine Cloud Create endpoint 1](./create-endpoint-1.webp)
   <center><figcaption>Figure 2. Create Endpoint</figcaption></center>
3. Select Other endpoint services.
4. Enter the service name that you choose in **Step 1**. Click **Verify service**.
5. Select your VPC in the drop-down list. 
6. In the Subnets area, select all the availability zones, and select the Subnet ID.
7. Select your security group properly in the Security groups area.
   :::note
   Make sure the selected security group allows inbound access from your EC2 instances on port 443.

   :::
8. Click Create endpoint. Then you have the **VPC endpoint ID**.

### Step 3: Create endpoint connection using TDengine Cloud

> 1. In TDengine Cloud left navigation pane, select **Network**, then choose the **PrivateLink** tab, click **Create Connection** in the upper-right corner. The **Create Connection** page is displayed.

Reviewer: ```suggestion
1. In TDengine Cloud left navigation pane, select **Network**, then choose the **PrivateLink** tab, click **Add New Private Link** in the upper-right corner. The **Add New Private Link** page will be displayed.
```

> 2. Firstly, set your preferred connection name, select the correct endpoint service to use to create the connection, and then enter the **VPC endpoint ID** you created in step 2.

Reviewer: ```suggestion
2. Firstly, set your preferred connection name, select the correct endpoint service to use to create the connection, and then enter the **Endpoint ID** you created in step 2.
```

> 3. Click the **Validate** button to verify if the request existed and could be accepted. 

Reviewer: ```suggestion
3. Click the **Verify** button to verify if the request existed and could be accepted. 
```

> 4. Click the **Confirm** button to create the private endpoint connection.

Reviewer: ```suggestion
4. Click the **Add** button to create the private endpoint connection.
```

5. you can find the connection info in the connection list in PrivateLink page.
6. Wait a few minutes, then refresh the page to see the connection status is CONNECTED.
7. The connection have three status: CONNECTED, DISCONNECTED and PENDING. When the operation is in progress, the connection status is PENDING. You need to wait a few minutes for the operation to complete.

### Step 4: Enable private DNS names in AWS

> 1. Click the endpoint id link in **Endpoints** page you created in AWS the Step 2.

Reviewer: ```suggestion
1. Click the endpoint id link in **Endpoints** page you created Step 2 on AWS.
```

2. Click **Actions**  in the upper-right corner and then select **Modify private DNS name**.
3. Check the box **Enable for this endpoint** and then click **Save changes**.
4. Then you can find the **Private DNS names** shown in **Endpoint Details** page.

### Step 5: Using the private DNS name to call TDengine Cloud Service

> Now you can access TDengine Cloud service in your VPC using the private DNS name. You can find the **private URL** in the [Instance page](https://console.cloud.tdengine.com/instances) in TDengine Cloud.

Reviewer: ```suggestion
Now you can access TDengine Cloud service in your VPC using the private DNS name. You can find the **Private URL** in the [Instances page](https://console.cloud.tdengine.com/instances) in TDengine Cloud.
```

## How to remove the endpoint connection

> 1. Disconnect the connection in **endpoint connection list** page in TDengine Cloud. Wait the connection status to be DISCONNECTED.

Reviewer: ```suggestion
1. Click the Action button in **endpoint connection list** page in TDengine Cloud. After a while, the connection status will be changed into DISCONNECTED.
```
